README.md

```diff
@@ -83,6 +83,12 @@ These are available in `/sequence`:
 - `/transformsInMaterializer` shows how connecting two subgraphs can invoke transformations in between. For example, the `city` of a customer can be fed into a `weather` that takes `lat,lon` by calling some geocoding in the sequence
 - `/useOfJSON` shows how, in long sequences, a new type does not have to created for every step--treat everything as JSON up to the last step, and your type system stays clean.
 
+## pagination
+
+For more information on pagination for REST or Databases, [see our documentation](https://www.ibm.com/docs/en/api-connect/ace/saas?topic=service-paginating-rest-api-as-data-source).
+
+- `/cursor` shows how to implement cursor based pagination for a database.
```

Comment:
Maybe best to call this database pagination, not `cursor`, and then move it to be `dbquery/pagination`.